Makefile

```diff
@@ -16,6 +16,7 @@ BASE_DIR := $(shell git rev-parse --show-toplevel)
 
 BIN :=$(BASE_DIR)/bin
 GOFMT := gofmt
++GLIDEPATH := $(shell command -v glide 2> /dev/null)
```

Comment:
what does the + mean?

Reply:
I had picked this up directly from https://github.com/lpabon/quartermaster/blob/dev/Makefile#L13

Doesn't appear this is needed. Removed it.